integration_tests/dbt_project.yml

```diff
@@ -65,16 +65,17 @@ vars:
     creative_url_tag_history: "{{ ref('snapchat_creative_url_tag_history_data') }}"
 
   tiktok_ads_source:
-    ad_account:  "{{ ref('tiktok_advertiser_data') }}"
     ad_history: "{{ ref('tiktok_ad_history_data') }}"
     ad_report_hourly: "{{ ref('tiktok_ad_report_hourly_data') }}"
     ad_group_history: "{{ ref('tiktok_adgroup_history_data') }}"
-    ad_group_report_hourly: "{{ ref('tiktok_adgroup_report_hourly_data') }}"
+    ad_group_hourly: "{{ ref('tiktok_adgroup_report_hourly_data') }}"
```

Comment:
thanks for catching this part... I need to update this in the transforms as well!